apollo-federation/src/operation/rebase.rs

```diff
@@ -276,15 +275,17 @@ impl Field {
         else {
             return Ok(None);
         };
-        if let Ok(federation_spec_definition) = get_federation_spec_definition_from_subgraph(schema)
```

Comment:
Refactors:
- I forgot when writing this that we could just use the subgraph metadata in the schema, I've updated it accordingly.